frontend/src/pages/modelRegistrySettings/CreateModal.tsx

```diff
@@ -11,21 +11,22 @@ import {
   TextInput,
   Title,
 } from '@patternfly/react-core';
-import './CreateModal.scss';
 import PasswordInput from '~/components/PasswordInput';
 import DashboardModalFooter from '~/concepts/dashboard/DashboardModalFooter';
 import { ModelRegistryKind } from '~/k8sTypes';
 import { MODEL_REGISTRY_DEFAULT_NAMESPACE } from '~/concepts/modelRegistry/const';
 import { ModelRegistryModel } from '~/api';
-import { createModelRegistryBackend } from '~/services/modelRegistryService';
+import { createModelRegistryBackend } from '~/services/modelRegistrySettingsService';
 import { isValidK8sName } from '~/concepts/k8s/utils';
+import './CreateModal.scss';
 
 type CreateModalProps = {
   isOpen: boolean;
   onClose: () => void;
+  refresh: () => Promise<unknown>;
 };
 
-const CreateModal: React.FC<CreateModalProps> = ({ isOpen, onClose }) => {
+const CreateModal: React.FC<CreateModalProps> = ({ isOpen, onClose, refresh }) => {
```

Comment:
Mmmmm didn't comment this before, and it's not a bad addition cause in the admin panel we were using old not refactored forms, but we got `useGenericObjectState` for creating objects and using them for forms.

We should refactor this modal in the near future with a custom hook that feeds from `useGenericObjectState`